vizro-core/src/vizro/tables/dash_table.py

```diff
@@ -0,0 +1,36 @@
+"""Module containing the standard implementation of `dash_table.DataTable`."""
+from collections import defaultdict
+from collections.abc import Mapping
+from typing import Any
+
+import pandas as pd
+from dash import dash_table
+
+from vizro.models.types import capture
+
+
+def _set_defaults_nested(supplied: Mapping[str, Any], defaults: Mapping[str, Any]) -> dict[str, Any]:
+    supplied = defaultdict(dict, supplied)
+    for default_key, default_value in defaults.items():
+        if isinstance(default_value, Mapping):
+            supplied[default_key] = _set_defaults_nested(supplied[default_key], default_value)
+        else:
+            supplied.setdefault(default_key, default_value)
+    return dict(supplied)
+
+
+@capture("table")
+def dash_data_table(data_frame: pd.DataFrame, **kwargs):
+    """Standard `dash_table.DataTable`."""
+    defaults = {
```

Comment:
Possibly we'll want to make this `defaults` publicly accessible also so that people can easily use it in their custom table functions, but no rush to do that yet.